backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/UserPermission.java

```diff
@@ -0,0 +1,15 @@
+package gov.cdc.usds.simplereport.config.authorization;
+
+/**
+ * Permissions that a user may hold, usually via their {@link OrganizationRole}.
+ */
+public enum UserPermission {
+    READ_PATIENT_LIST,
+    READ_RESULT_LIST,
+    EDIT_PATIENT,
```

Comment:
I have been assuming that the `EDIT_PATIENT` permission includes adding a patient as part of its scope, tho I can see a world where ADD_PATIENT may warrant its own permission (might be overly specific). 

Same with UPDATE_TEST to include deleting tests from the test queue.

Can't currently delete patients atm, but that may be a thing?

Reply:
Deleting patients will be an admin-only function. I concur that creating and editing patients are the same action, unless and until we get a different requirement.